src/installJava.ts

```diff
@@ -0,0 +1,110 @@
+import * as path from "path";
+import fetch from "node-fetch";
+import * as fs from "fs";
+import mkdirp = require("mkdirp");
```

Comment:
Could you also have just done `import * as mkdirp from "mkdirp";`?

Reply:
Not really, since mkdirp is not a namespace, but it’s a function exported as a commonjs module

Reply:
There’s a compatibility mode for TS that I may have used, but it doesn’t change much

Reply:
👍